src/steps/git.rs

```diff
@@ -364,6 +368,12 @@ impl RepoStep {
             return Ok(());
         }
 
+        let verbose = ctx.config().verbose();
+
+        if !verbose {
+            println!("\n{} updated repositories will be shown...\n", style("Only").green().bold());
+        }
```

Comment:
I kinda think it is unnecessary to inform users of this on every run, thoughts?

Reply:
I could go either way. It's somewhat helpful if you have a lot of repositories to know that something is happening.

Reply:
> It's somewhat helpful if you have a lot of repositories to know that something is happening.

i agree